app/__init__.py

```diff
@@ -6,7 +6,7 @@
 from time import monotonic
 
 from dotenv import load_dotenv
-from flask import _request_ctx_stack, g, jsonify, make_response, request
+from flask import _request_ctx_stack, g, jsonify, make_response, request  # type: ignore
```

Comment:
`_request_ctx_stack` is missing from the type definitions for flask for some reason